### PR TITLE
docs full page mastheads toolbar formatting fix

### LINKED ISSUE #8797: v5 - Mastheads in full page demos are not formatted correctly
The following demos have weird masthead formatting:

- [ ] [AlertGroup](http://patternfly-react-v5.surge.sh/components/alert-group/react-demos/alert-group-toast-with-notification-drawer/)
Screenshot 2023-03-09 at 11 48 04 AM

- [ ] http://patternfly-react-v5.surge.sh/components/notification-drawer/react-demos/basic/
Screenshot 2023-03-09 at 11 48 22 AM

I didn't do a full audit - I noticed the first of the full page Table demos I looked at work formatted correctly. I'm not sure if this is a wide spread issue with the dashboard wrapper or specific to those demos.

### PR DESCRIPTION
**Pull request by MariaAga**
**What**: Closes #8797 

Checked all other full page demos and they look fine

### CODE REVIEW

File: packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ class BasicNotificationDrawer extends React.Component {
       </DropdownGroup>
     ];
     const headerToolbar = (
-      <Toolbar>
+      <Toolbar isFullHeight>
```
Comment (nicolethoen) [inline packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: I believe the Dropdowns themselves also need the isFullHeight modifier on them to make the height of the user dropdowns consistent between examples

Comment (nicolethoen) [reply packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: I think that's like line 874

Comment (MariaAga) [reply packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: It does look to make any visual change and 7/12 examples for username dont have that (searched for `ned username` and `john smith`) so should we change all example to have it or remove it from existing examples?

**Review by thatblindgeye — APPROVED** [review]
Other than the above convo, looks good!

File: packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md (hunk 1 above)
Comment (nicolethoen) [reply packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: @mcarrano 
Should the user dropdown in mastheads in our demos always be full height? Looks like in the react workspace right now, only half of them are full height.

Comment (mcarrano) [reply packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: Yes, I believe so, i.e. when they are open, the selected highlight is at the bottom of the masthead.

Comment (MariaAga) [reply packages/react-core/src/demos/NotificationDrawer/NotificationDrawer.md:241]: Oh I see, I also counted the examples for `pageHeader`. 
Added `isFullHeight`

**Review by mcarrano — APPROVED** [review]
Looks good to me. Thanks @MariaAga

**Review by kmcfaul — APPROVED** [review]

**Pull request merged**